CHANGELOG.md

```diff
@@ -1,4 +1,26 @@
 
+1.2.2 / 2021-07-08
+==================
+
+This release contains various bugfixes, some useful features and important performance improvements.
+It also uses Kotlin 1.5.20 as default.
+
+### Features
+
+  * Support for `@JsonNames` and `coerceInputValues` in Json.decodeFromDynamic (#1479)
+  * Add factory function to wrap a serial descriptor with a custom name for custom delegating serializers (#1547) (thanks to [Fadenfire](https://github.com/Fadenfire))
+  * Allow contextually serialized types to be used as map keys in Json (#1552) (thanks to [pdvrieze](https://github.com/pdvrieze))
+
+### Bugfixes and performance improvements
+
+  * Update size in JsonStringBuilder slow-path to avoid excessive array-copies for large strings with escape symbols (#1491)
+  * Optimize CBOR Integer encoding length (#1570) (thanks to [davertay](https://github.com/davertay))
+  * Throw JsonDecodingException instead of ClassCastException during unexpected null in TreeJsonDecoder (#1550)
```

Comment:
[nit] You're using backticks for symbols in `Features` section, but not here. It's worth being consistent